src/Components/Navbar/Navbar.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import './Navbar.css'
+import { Link } from "react-router-dom"
+
+function Navbar() {
+  return (
+    <div className="top-bar">
+        <div className="left-topbar-container">
+            {/* <button id="top-bar-ham-menu-btn" className="icon-btn"><i className="fa fa-bars" aria-hidden="true"></i></button> */}
+            <Link to="/">
+                <h2 id="top-bar-brand-name">Bookztron</h2>
+            </Link>
+            <div className="search-bar">
+                <input className="search-bar-input" placeholder="Search"/>
+                <button id="search-bar-btn">
+                    <i className="fa fa-search" aria-hidden="true"></i>
+                </button>
+            </div>
+        </div>
+        <div className="right-topbar-container">
+            <button id="navbar-login-btn" className="solid-primary-btn">Login</button>
+            <button id="my-wishlist-btn" className="icon-btn">
+                <div className="icon-count-badge">
+                    <i className="fa fa-heart-o fa-x" aria-hidden="true" ></i>
+                    <span className="count-badge-x">4</span>
+                </div>
+            </button>
+            <button id="my-cart-btn" className="icon-btn">
+                <div className="icon-count-badge">
+                    <i className="fa fa-shopping-cart fa-x" aria-hidden="true" ></i>
+                    <span className="count-badge-x">2</span>
+                </div>
+            </button>
+        </div>
+    </div>
+  )
```

Comment:
Suggestion: id attribute could be removed from buttons

Reply:
Yeah, will need to update Component Library for this one, will do it.